lib/Dialect/FIRRTL/Transforms/LowerTypes.cpp

```diff
@@ -171,10 +173,13 @@ static bool peelType(Type type, SmallVectorImpl<FlatBundleFieldEntry> &fields,
   // then just return.
   if (isPreservableAggregateType(type, mode))
     return false;
-
+  FIRRTLBaseType base;
   if (auto refType = type.dyn_cast<RefType>())
-    type = refType.getType();
-  return TypeSwitch<Type, bool>(type)
+    base = refType.getType();
+  else
+    base = type.cast<FIRRTLBaseType>();
+
+  return TypeSwitch<Type, bool>(base.getAnonymousType())
```

Comment:
This is going to strip names recursively rather than iteratively as the function worked before.  This will cause changes to bundle<a: alias<int> > which are unnecessary.